assets/methods_description_template.yml

```diff
@@ -0,0 +1,26 @@
+id: "nf-core-phyloplace-methods-description"
+description: "Suggested text and references to use when describing pipeline usage within the methods section of a publication."
+section_name: "nf-core/phyloplace Methods Description"
+section_href: "https://github.com/nf-core/phyloplace"
+plot_type: "html"
+data: |
+  <h4>Methods</h4>
+  <p>Data was processed using nf-core/phyloplace v${workflow.manifest.version} ${doi_text} of the nf-core collection of workflows (<a href="https://doi.org/10.1038/s41587-020-0439-x">Ewels <em>et al.</em>, 2020</a>).
+  Briefly, the workflow aligns a number of query sequences to a reference alignment using either HMMER (<a href="http://dx.doi.org/10.1371/journal.pcbi.1002195">Eddy 2011</a>) or MAFFT (<a href="https://doi.org/10.1093/nar/gkf436">Katoh <em>et al.</em> 2002) and then places the query sequences in a reference tree using EPA-NG (<a href="https://doi.org/10.1093/sysbio/syy054">Barbera <em>et al.</em> 2019</a>).
+  Subsequently, Gappa (<a href="https://doi.org/10.1093/bioinformatics/btaa070">Czech <em>et al.</em> 2020) is used to produce a newick file with query sequences grafted on to the reference tree.
+  </p>
+  <p>The pipeline was executed with Nextflow v${workflow.nextflow.version} (<a href="https://doi.org/10.1038/nbt.3820">Di Tommaso <em>et al.</em>, 2017</a>) with the following command:</p>
+  <pre><code>${workflow.commandLine}</code></pre>
+  <h4>References</h4>
+  <ul>
+    <li>Di Tommaso, P., Chatzou, M., Floden, E. W., Barja, P. P., Palumbo, E., & Notredame, C. (2017). Nextflow enables reproducible computational workflows. Nature Biotechnology, 35(4), 316-319. <a href="https://doi.org/10.1038/nbt.3820">https://doi.org/10.1038/nbt.3820</a></li>
+    <li>Ewels, P. A., Peltzer, A., Fillinger, S., Patel, H., Alneberg, J., Wilm, A., Garcia, M. U., Di Tommaso, P., & Nahnsen, S. (2020). The nf-core framework for community-curated bioinformatics pipelines. Nature Biotechnology, 38(3), 276-278. <a href="https://doi.org/10.1038/s41587-020-0439-x">https://doi.org/10.1038/s41587-020-0439-x</a></li>
+  </ul>
```

Comment:
Here typically you can add the citations that you have in the text above linked.

Reply:
Thanks, done.